lib/report_portal/cucumber/report.rb

```diff
@@ -114,8 +115,8 @@ def test_step_finished(event, desired_time = ReportPortal.now)
         end
       end
 
-      def done(desired_time = ReportPortal.now)
-        end_feature(desired_time) if @feature_node
+      def test_run_finished(_event, desired_time = ReportPortal.now)
```

Comment:
This change is related to https://github.com/reportportal/agent-ruby/pull/45 but it was done here to prevent a conflict

Reply:
#45 was merged. Resolve the issue.